src/event.rs

```diff
@@ -86,7 +95,7 @@ pub enum Event {
     /// Our own section has been split, resulting in the included `Prefix` for our new section.
     SectionSplit(Prefix<XorName>),
     /// The client has successfully connected to a proxy node on the network.
-    Connected,
+    Connected(ConnectEvent),
```

Comment:
it is referred as `connect_type` in the later on Debug impl .
Also, `ConnectType` seems reflects the usage more accurately?

Reply:
My preference would be to have it as separate events: `Event::Connected` and `Event::Relocated`. We discussed it with @jeanphilippeD  and decided it's something that can be addressed later.